Source/src/modules/ModuleRender.cpp

```diff
@@ -93,24 +93,24 @@ void Hachiko::ModuleRender::GenerateFrameBuffer()
     glBindFramebuffer(GL_FRAMEBUFFER, 0);
 }
 
-void Hachiko::ModuleRender::ResizeFrameBuffer(int heigth, int width) const
+void Hachiko::ModuleRender::ResizeFrameBuffer(const int width, const int height) const
```

Comment:
As @Bernatmago mentioned, const does not do anything here since the value passed to the function is a copy of the original.

Reply:
thank you, it's both for self documenting and assumptions